drive/drive.go

```diff
@@ -0,0 +1,113 @@
+package drive
+
+import (
+	"context"
+
+	logging "github.com/ipfs/go-log"
+
+	fxiface "github.com/functionland/go-fula/fxfs/core/iface"
+	"github.com/functionland/go-fula/fxfs/core/pfs"
+	files "github.com/ipfs/go-ipfs-files"
+	"github.com/ipfs/interface-go-ipfs-core/path"
+)
+
+var log = logging.Logger("fula-drive")
+
+type Drives struct {
+	Drives []UserDrive `json:"drives"`
+}
+
+type UserDrive struct {
```

Comment:
I recommend getting into the habit of writing godoc as we go along. All exported types should have documentation. 

Please feel free to do this in a separate PR.